Dockerfile.rhel8

```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8
```

Comment:
I would rather name these rhel based ones call Dockerfile, as it's default and production variant. And I would add centos suffix to dockerfiles based on centos because there are only for development purposes.